presto-spark-base/src/test/java/com/facebook/presto/spark/PrestoSparkQueryRunner.java

```diff
@@ -339,6 +339,12 @@ public PrestoSparkQueryRunner(
         // Install tpch Plugin
         pluginManager.installPlugin(new TpchPlugin());
         connectorManager.createConnection("tpch", "tpch", ImmutableMap.of());
+        // TPCH-Standard uses tpch column naming
+        // See: https://github.com/prestodb/presto/issues/1771
+        Map<String, String> tpchProperties = ImmutableMap.<String, String>builder()
+                .put("tpch.column-naming", "standard")
+                .build();
+        connectorManager.createConnection("tpchstandard", "tpch", tpchProperties);
```

Comment:
I wonder if this is the only change that's really necessary.

Reply:
Actually, this works on detached mode. For non-detached mode, it requires some more change. Working on it.

Reply:
This was not working in non-detached mode as the connectorManager has hardcoded connector names. I have refactored how NativeExecutionConnectionManager is injected.

Reply:
one more change pending, i will close this PR and re-open when it is pushed.